redpanda/assets/dashboards/overview.json

```diff
@@ -1,12 +1,12 @@
 {
   "title": "Redpanda Overview",
-  "description": "## Title\n\nDescribe this dashboard. Add links to other dashboards, monitors, wikis,  and docs to help your teammates. Markdown is supported.\n\n- [This might link to a dashboard](#)\n- [This might link to a wiki](#) (cloned)",
+  "description": "## Title\n\nDescribe this dashboard. Add links to other dashboards, monitors, wikis,  and docs to help your teammates. Markdown is supported.\n\n- [This might link to a dashboard](#)\n- [This might link to a wiki](#) (cloned) (cloned)",
   "widgets": [
     {
       "id": 1004651040127398,
       "definition": {
         "type": "image",
-        "url": "https://raw.githubusercontent.com/DataDog/integrations-extras/master/redpanda/images/redpanda_horizontal_logo.png",
+        "url": "https://7733588.fs1.hubspotusercontent-na1.net/hubfs/7733588/Redpanda_Logo_Primary%20(2).png",
```

Comment:
@tmgstevens You might want to use a different source for the logo, like what was previously referenced which was in this github repo.

Reply:
This is our official logo source